README.md

```diff
@@ -1 +1,87 @@
-# to-do-list
\ No newline at end of file
+<br><br><br>
+
+# To-Do List
+
+to-do list version 1.0.0
+> list structure
+
+<br>
+<br>
+
+## Built With
+<hr><br>
+
+> GitBash
+
+> HTML/CSS
+
+> Javascript
+
+> Webpack
+>> HTMLWebpackPlugin<br>
+>> WebpackDevServer<br>
+>> CSS Loader<br>
+>> Style Loader
+
+
+<br><br>
+
+## Live Demo (if available)
+<hr>
+
+Click [here](https://livedemo.com) to view live demo link.
+<br><br>
+
+## Setup
+<hr>
+
+To get a local copy up and running, follow these steps:
+1. **Click "Code" button**
+2. Select HTTPS clone method and [**copy Repo URL**](https://github.com/Botlhale-Setou/to-do-list.git)
+3. **Open Git Bash** (or other terminal) and **enter "cd [Local Workstation Path]"**
+4. **Enter "git clone [Repo URL](https://github.com/Botlhale-Setou/to-do-list.git)"**
+5. **Enter "git checkout -b [New-Branch-Name]"**
+<br><br><br>
+## Usage
+<hr>
+A simple to-do list web application to help track tasks.
+<br><br>
+
+## Run tests
+Linters.
+<br>
+
+## Deployment
+Mobile and desktop browsers
+<br><br><br>
+
+## Authors
+<hr><br>
+
+👤 **Botlhale Setou**
+
+- GitHub: [@Botlhale-Setou](https://github.com/Botlhale-Setou)
+- Twitter: [@Botlhale_Setou](https://twitter.com/Botlhale_Setou)
+- LinkedIn: [@botlhalesetou](https://www.linkedin.com/in/botlhalesetou/)
+<br><br><br>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Botlhale-Setou/to-do-list/issues).
+<br><br>
+## Show your support
+
+Give a ⭐️ if you like this project!
+<br><br>
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+<br><br><br>
+## 📝 License
+<hr>
+
+This project is [MIT](./dist/main.js.LICENSE.txt) licensed.
```

Comment:
- [ ] The link to the MIT license is broken. Please fix it as it is very important 👍

Reply:
@codepantha link resolved as well 👍 
<be>

![image](https://user-images.githubusercontent.com/11477535/193107976-c9d6a63c-8807-4b70-8805-6b8d8f895c65.png)